<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorSettings</class>
 <widget class="QDialog" name="EditorSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>265</width>
    <height>203</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pbOK">
   <property name="geometry">
    <rect>
     <x>94</x>
     <y>170</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbFontSize">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>89</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Front size</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbTheme">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>89</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Theme</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbSplitScreenView">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Split screen view</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
